R/summarise.R

```diff
@@ -121,22 +121,55 @@ summarise.data.frame <- function(.data, ...) {
 }
 
 #' @export
-summarise.grouped_df <- function(.data, ...) {
+summarise.grouped_df <- function(.data, ..., .groups = NULL) {
   out <- NextMethod()
 
   group_vars <- group_vars(.data)
-  n <- length(group_vars)
-  if (n > 1) {
-    out <- grouped_df(out, group_vars[-n], group_by_drop_default(.data))
+  if (is.null(.groups) || identical(.groups, "drop_last")) {
+    n <- length(group_vars)
+    if (n > 1) {
+      if (is.null(.groups)) {
+        inform(c(
+          glue("summarised tibble has groups: {new_groups}", new_groups = glue_collapse(group_vars[-n], sep = ", ")),
+          i = 'This message can be turned off using: summarise(..., .groups = "drop_last")'
```

Comment:
```suggestion
          i = 'Turn off this message by with `summarise(..., .groups = "drop_last")`'
```
?

But I think ideally this message would fit on one line? Maybe:

```R
Grouping by 'mpg', 'cyl', 'vs' (`.groups = "drop_last"`)
```

?